test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -158,4 +160,53 @@ public void handleSingleNodeFailure()
         assertThat(l2.getLayoutServers().size()).isEqualTo(2);
         assertThat(l2.getLayoutServers().contains(SERVERS.ENDPOINT_1)).isFalse();
     }
+
+    /***
+     * Scenario with 1 node: SERVERS.PORT_0
+     * The node is setup, bootstrapped and then requested for a
+     * heartbeat. This is responded with the nodeMetrics which is
+     * asserted with expected values.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void checkHeartbeat()
```

Comment:
I'll add a few negative tests.
Cannot add multi-node test as there is no policy in place (in this PR) to have communication among nodes using heartbeats.